ACPs/131-cancun-eips/README.md

```diff
@@ -0,0 +1,59 @@
+| ACP | 131 |
+| :--- | :--- |
+| **Title** | Activate Cancun EIPs on C-Chain and Subnet-EVM chains |
+| **Author(s)** | Darioush Jalali ([@darioush](https://github.com/darioush)), Ceyhun Onur ([@ceyonur](https://github.com/ceyonur)) |
+| **Status** | Proposed ([Discussion](POPULATED BY MAINTAINER, DO NOT SET)) |
+| **Track** | Standards, Subnet |
+
+## Abstract
+
+Enable new EVM opcodes and opcode changes in accordance with the following EIPs on the Avalanche C-Chain and Subnet-EVM chains:
+- [EIP-4844: BLOBHASH opcode](https://eips.ethereum.org/EIPS/eip-4844)
```

Comment:
I think the abstract should also call out that blobs are blocked in `coreth`. I know that it's mentioned below but some people might only read the abstract.

Reply:
updated